libr/anal/reflines.c

```diff
@@ -476,7 +476,8 @@ R_API char* r_anal_reflines_str(void *_core, ut64 addr, int opts) {
 		}
 	}
 	add_spaces (b, 0, pos, wide);
-	str = r_buf_free_to_string (b);
+	str = r_buf_to_string (b);
```

Comment:
What about r_buf_drain()

Reply:
Yes we *could* add it, but we need to keep a separate `r_buf_to_string` anyway. Moreover, we would need a `r_buf_drain_string` and a `r_buf_drain`. I'm not sure for now. Can we keep it this way for now and maybe think about it as we work with it?